packages/core/src/components/Tabs/Tabs.test.jsx

```diff
@@ -22,6 +22,19 @@ function shallowRender(customProps = {}, children) {
   };
 }
 
+function deepMount(customProps = {}, children) {
```

Comment:
Nit: Rather than creating a new method for using `mount` rather than `shallow`, we can follow the same [pattern we use in `VerticalNavItem.test.jsx`](https://github.com/CMSgov/design-system/blob/master/packages/core/src/components/VerticalNav/VerticalNavItem.test.jsx#L5-L16), where it supports passing in a `deep` boolean argument which determines whether `shallow` (default) or `mount` is used.